telegram/ext/_applicationbuilder.py

```diff
@@ -928,6 +931,42 @@ async def post_init(application: Application) -> None:
         self._post_init = post_init
         return self
 
+    def post_shutdown(
+        self: BuilderType, post_shutdown: Callable[[Application], Coroutine[Any, Any, None]]
+    ) -> BuilderType:
+        """
+        Sets a callback to be executed by :meth:`Application.run_polling` and
+        :meth:`Application.run_webhook` *after* executing :meth:`Application.shutdown` but
+        *before* executing :meth:`Updater.shutdown` respectively.
```

Comment:
```suggestion
        :meth:`Application.run_webhook` *after* executing :meth:`Updater.shutdown`
        and :meth:`Application.shutdown`.
```

Reply:
Oops, missed that comment

Is this up to me to commit? If yes let me know, i don't want to hold this PR just for this (sorry)

Reply:
yeah, you can just commit my suggestion (click the button above)